packages/cli/src/config/schema.ts

```diff
@@ -35,6 +35,12 @@ export const schema = {
 				default: -1,
 				env: 'N8N_CONCURRENCY_PRODUCTION_LIMIT',
 			},
+			evaluationLimit: {
+				doc: 'Max evaluation executions allowed to run concurrently.',
```

Comment:
Let's add the other details as well? They all apply.

Reply:
In queue mode, concurrency control uses the exact same env but is [delegated to Bull](https://github.com/OptimalBits/bull/blob/master/REFERENCE.md#queueprocess).